src/wam2layers/utils/profiling.py

```diff
@@ -47,14 +48,21 @@ def store_intermediate_states(self, output):
         don't lose accumulations from previous outputs.
         """
         totals = output.sum()
+
         if self.mode == "backtrack":
             self.tracked += totals["e_track"]
             self.total_tagged_moisture += totals["tagged_precip"]
         else:  # mode is forwardtrack
             self.tracked += totals["p_track_lower"] + totals["p_track_upper"]
             self.total_tagged_moisture += totals["tagged_evap"]
+
+        # Don't include boundary losses in log messages
+        interior_losses = output.losses[1:-1, 1:-1].sum()
```

Comment:
As you use a simple mask to split interior/boundary, I guess this means that you can theoretically overestimate "boundary losses" and underestimate interior losses :thinking: 
But that's probably more of an academic point and doesn't really matter in practice.

Reply:
I rather include both in the log messages, so that a user can see what's happening.